pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -107,6 +107,11 @@ let
       CLS_U32_PERF       = yes;
       CLS_U32_MARK       = yes;
       BPF_JIT            = mkIf (stdenv.hostPlatform.system == "x86_64-linux") yes;
+      BPF_JIT_ALWAYS_ON  = mkIf (stdenv.hostPlatform.system == "x86_64-linux") yes;
```

Comment:
arm and arm64 should have BPF JIT since quite some time, too.

Reply:
they have. is there a sane way to implement the check? ( like platform.oneOf [ "arm", "arm64", ... ]?

Reply:
I'd use `stdenv.hostPlatform.isAarch64` and `stdenv.hostPlatform.isAarch32`, if I get correctly what you mean.